.github/workflows/issues-close-project-done.yaml

```diff
@@ -0,0 +1,27 @@
+name: Project Issue State Sync
+on:
+  schedule:
+    # Runs on minute 0 every 2 hours
+    - cron: 0 0-23/2 * * *
+  workflow_dispatch:
+  # Manual trigger
+jobs:
+  issue-state-sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Sync issues for dedupe
+        uses: dasmerlon/project-issue-state-sync@v2
+        with:
+          github_token: ${{ secrets.CLOSE_ISSUE_WHEN_PROJECT_DONE }} #this custom action doesn't allow the default GITHUB_TOKEN, it requires us to use a PAT as the secret unfortunetly..
+          owner: CDCgov
+          project_number: 34
+          closed_statuses: Done
+          open_statuses: Todo,In Progress,Review
```

Comment:
Do these columns need to be explicitly stated?  Just wondering if we decide to add another column to the board will we also have to remember to update this workflow?

Reply:
No they shouldn't. Whoever made this action, configured it so `open_statuses` is `required: false`. It looks like its just used for log handling https://github.com/dasmerlon/project-issue-state-sync/blob/cd5548299b1e2c64b31448ec742f0d5317b5f175/src/main.rs#L34

Reply:
Actually, I take that back it is use for checking state but seems like it just gets ignored if its not labeled which SHOULD just leave it open.

Reply:
In that case, what do you think about removing `open_statuses`

Reply:
I'm for it. I'm going to test the action without it first.

Reply:
Worked as expected.